webapp/src/redux/ducks/studyDataSynthesis.ts

```diff
@@ -92,6 +112,20 @@ export const setStudyData = createAsyncThunk<
   return api.getStudySynthesis(id as string).catch(rejectWithValue);
 });
 
+export const updateStudyData =
+  (event: WSMessage<GenericInfo>): AppThunk =>
+  (dispatch, getState) => {
+    const state = getState();
+    if (
+      studyDataAdapter
```

Comment:
Add `export const getStudyDataIds = studyDataSelectors.selectIds` in selectors.ts instead